Mathematica.java

```diff
@@ -2,4 +2,33 @@
     public static int sum(int a, int b){
         return a + b;
     }
+    public static int power (int a, int b) {
+        int result = a;
+        for (int i = 1; i <= b; i++){
+            result *= a;
+        }
+        return result;
+    }
+    public static int subtract(int a, int b) {
+        return a - b;
+    }
+    public static int multiplication(int a, int b){
+        return a * b;
+    }
+    public static int division(int a, int b){
+        return a / b;
+    }
```

Comment:
Method `Abs` should follow standard method naming convention. Should be `abs(int x)`